model/load_balancer.rb

```diff
@@ -4,6 +4,7 @@
 
 class LoadBalancer < Sequel::Model
   many_to_many :vms
+  many_to_many :vms_to_dns, class: :Vm, left_key: :load_balancer_id, right_key: :vm_id, join_table: :load_balancers_vms, conditions: Sequel.lit("state NOT IN ('evacuating')")
```

Comment:
Does this work instead of Sequel.lit?

```ruby
{state: Sequel.exclude(['evacuating'])}
```

Reply:
I would like to keep it. It makes reading the code easier.